packages/blocks/jira-cloud/src/lib/common/index.ts

```diff
@@ -73,14 +73,58 @@ export async function searchUserByCriteria(
   return response.body.length === 0 ? [] : (response.body as JiraUser[]);
 }
 
+function shouldFetchNextPage(args: {
+  valuesCount: number;
+  total?: number;
+  isLast?: boolean;
+  startAt: number;
+  maxResults: number;
+}): boolean {
```

Comment:
https://docs.atlassian.com/software/jira/docs/api/REST/1000.1609.0/a

pagination with jira is either very confusing or im very confused, total is optional, islast is optional but somehow we should still check them. idk let me know your thoughts